### PR TITLE
adding resources page

### PR DESCRIPTION
**Pull request by rubyimrie**
adding resources page, no big changes

### CODE REVIEW

File: pages/resources.vue
Comment (Sephster) [inline pages/resources.vue]: This page feels a bit sparse. Would it be worth adding either some text at the top saying what this page offers and then providing the links as a list or sectioning each link and adding a bit of supporting text saying what the resource is? Cheers Ruby

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/resources.vue
Comment (rubyimrie) [reply pages/resources.vue]: have added in an introduction sentence and a descripting sentence for each source. Have also deleted the paragraph and link styling as this is now done globally so will appear as such when merged in.

Comment (Sephster) [reply pages/resources.vue]: ahh cool. Thanks Ruby. I'll keep this issue open and once we've merged in the link and paragraph styles, I'll take another look at this to make sure all is ok but seems like the correct approach. Thank you!

**Review by Sephster — APPROVED** [review]
This looks good to go. Will need one final review but tentatively approved

File: pages/resources.vue
Comment (Sephster) [reply pages/resources.vue]: paragraphs usually have a bottom margin so there is a gap between paragraphs. Are we able to add this please? Otherwise, looks good. Thanks Ruby

Comment (rubyimrie) [reply pages/resources.vue]: oh had removed mb-3 to p in the basic layout but strangely  doesn't seem to be working, so will add it back in

Comment (Sephster) [reply pages/resources.vue]: thanks Ruby. Looks good now. Merging in

**Pull request merged**
